public/templates/index.html

```diff
@@ -124,21 +124,21 @@ <h5>For Practioners:</h5>
                     <p>If you are a practitioner sign up to offer free assistance to those in need. All credentials will be verified. </p>
                     <ul>
                         <li>
-                            <img src="/assets/images/about-icon-01.png" alt="">
+                            <img src="/public/assets/images/images/about-icon-01.png" alt="">
```

Comment:
Looks like the `src` path is wrong here. 

You actually moved the image files to `/public/asset/images` but your source has it going to `/public/asset/images/images`.

Ditto all the relative paths in this commit.

Reply:
Done